api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -186,12 +193,13 @@
      * is equivalent to infinity; The operator hence will be attempted to be recovered indefinitely unless this value
      * is set to anything else.
      */
-    Attribute<Integer> RECOVERY_ATTEMPTS = new Attribute<Integer>(new Integer2String());
+    Attribute<Integer> RECOVERY_ATTEMPTS = new Attribute<>(new Integer2String());
```

Comment:
Why is this the only place that changes <...> to <>?  Should we change the rest?